### PR TITLE
Fix images to use new url

### PR DESCRIPTION
**Pull request by jackmawer**
Updated the image urls for you.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # ZC-PB
 
-![UNLICENSE](https://cdn.rawgit.com/tomascw/tms-licenses/master/license-UNLICENSE-lightgrey.svg) ![CC0](https://cdn.rawgit.com/tomascw/tms-licenses/master/license-CC0-lightgrey.svg)
+![UNLICENSE](https://cdn.rawgit.com/cwverse/tms-licenses/master/license-UNLICENSE-lightgrey.svg) ![CC0](https://cdn.rawgit.com/cwverse/tms-licenses/master/license-CC0-lightgrey.svg)
```
Comment (jackmawer) [inline README.md]: - Updating from "tomascw" to "cwverse" in order to not rely on GitHub redirects

File: README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 
-![Zack Casey](https://cdn.rawgit.com/tomascw/zc-pb/master/z-portait.svg)
+![Zack Casey](https://cdn.rawgit.com/cwverse/zc-pb/master/src/z-portait.svg)
```
Comment (jackmawer) [inline README.md]: - Updating from "tomascw" to "cwverse" in order to not rely on GitHub redirects
- Adding "/src" to fix broken link

File: README.md (hunk 1 above)
Comment (tomascw) [inline README.md]: tms-licenses isn't apart of cwverse repo since I use it across all my other projects.

**Comment by tomascw** [comment]
Merged into single commit.

**Pull request merged**